src/com/trainee/phani/Diagonal.java

```diff
@@ -22,17 +22,18 @@ public static void main(String[] args) {
             }
             for (int i = 0; i < size; i++) {
                 for (int j = 0; j < size; j++) {
-                    if (i == j) {
+                    if (i == j) {    //to identify the left diagonal sum
```

Comment:
Comments should go above the if condition. Recommended and looks good

//to identify the left diagonal sum
if(i ==j)